source/elements/oneMKL/source/domains/spblas/format-descriptions.rst

```diff
@@ -7,16 +7,87 @@
+    and *val*, and an *index* parameter. Each non-zero
+    element, *i* in the sparse matrix is represented by its
+    row index, column index, and value, that is,
+    *(row_ind[i], col_ind[i], val[i])*. The entries need not
+    be in a sorted order.
+
+    .. container:: tablenoborder
+
+         .. list-table::
+
+            * - nrows
+              - Number of rows in the sparse matrix.
+            * - ncols
+              - Number of columns in the sparse matrix.
+            * - nnz
+              - Number of non-zero entries in the sparse matrix (which may include explicit zeros).
+                This is also the length of the *row_ind*, *col_ind* and *val* arrays.
+            * - index
+              - Parameter that is used to specify whether the matrix has zero or one-based indexing.
+            * - val
+              - An array of length ``nnz`` that contains the non-zero elements of the sparse matrix
+                not necessarily in any sorted order.
+            * - row_ind
+              - An integer array of length ``nnz``. Contains row indices for non-zero elements
+                stored in the *val* array such that *row_ind[i]* is the row number (using zero-
+                or one-based indexing) of the element of the sparse matrix stored in *val[i]*.
+            * - col_ind
+              - An integer array of length ``nnz``. Contains column indices for non-zero elements
+                stored in the *val* array such that *col_ind[i]* is the column number (using zero-
+                or one-based indexing) of the element of the sparse matrix stored in *val[i]*.
+
+
+A sparse matrix can be represented in a COO format in a following way (assuming one-based indexing):
+
+.. math::
+   A = \left(\begin{matrix}
+             1 &  0 & 2\\
+             0 & -1 & 4\\
+             3 &  0 & 0\\
+             \end{matrix}\right)
+
+
++------------+------------------------------------------------------------+
+| nrows      | 3                                                          |
++------------+------------------------------------------------------------+
+| ncols      | 3                                                          |
++------------+------------------------------------------------------------+
+| nnz        | 5                                                          |
++------------+------------------------------------------------------------+
+| index      | 1                                                          |
++------------+------------+-----------+-----------+-----------+-----------+
+| row_ind    | 1          | 1         | 2         | 2         | 3         |
++------------+------------+-----------+-----------+-----------+-----------+
+| col_ind    | 1          | 3         | 2         | 3         | 1         |
++------------+------------+-----------+-----------+-----------+-----------+
+| val        | 1          | 2         | -1        | 4         | 3         |
```

Comment:
Just a minor comment, in the CSR description, we present the arrays in the following order: `val` - `col_ind` - `row_ptr`. Do we want to follow the same order here? Or change it for CSR?
Or maybe each format should be presented in its own way that feels more "natural".

Reply:
Actually, I'd rather change the CSR description to match the order of the arguments in the spec. This looks good for the COO format.

Reply:
I did think of this actually. Eventually I kept CSR unchanged for minimizing the changes. I can make this change.

Reply:
If you think it's worth it. I'm fine if we keep it as is too

Reply:
I've made this change now.

Reply:
Thanks!